docs/pages/eas/workflows/syntax.mdx

```diff
@@ -237,6 +237,7 @@ Specific versions of tools that should be used for jobs defined in this workflow
 | ----------- | -------------------------------------------------------------------------- |
 | `node`      | Version of Node.js installed via `nvm`.                                    |
 | `yarn`      | Version of Yarn installed via `npm -g`.                                    |
+| `corepack`  | Whether to enable corepack.                                                |
```

Comment:
```suggestion
| `corepack`  | If set to `true`, [corepack](https://nodejs.org/api/corepack.html) will be enabled at the beginning of build process. Defaults to false.                                                |
```
I think developers might benefit to learn that it's a Boolean value and has a default value.